boards/PDM/Src/Io/Io_Efuse_Aux1Aux2.c

```diff
@@ -0,0 +1,131 @@
+#include "Io_Efuse_Aux1Aux2.h"
+#include "configs/Io_EfuseConfig.h"
+
+static struct Efuse_Context *aux1_aux2_efuse;
+
+void Io_Efuse_Init_Aux1Aux2(SPI_HandleTypeDef *const hspi)
+{
+    assert(hspi != NULL);
+
+    aux1_aux2_efuse = Io_Efuse_Create(
+        hspi, CSB_AUX1_AUX2_GPIO_Port, CSB_AUX1_AUX2_Pin,
```

Comment:
How do we currently "see" these GPIO constants? I don't see a `main.h` included so I was curious.

Reply:
The header for this file includes _Io_Efuse.h_, which includes _main.h_

Reply:
Oh tru

Reply:
Io_Efuse.h also needs it

Reply:
```
struct Efuse_Context *Io_Efuse_Create(
    SPI_HandleTypeDef *const hspi,
    GPIO_TypeDef *           chip_select_port,
    uint16_t                 chip_select_pin,
    GPIO_TypeDef *           fsob_port,
    uint16_t                 fsob_pin,
    GPIO_TypeDef *           fsb_port,
    uint16_t                 fsb_pin,
    GPIO_TypeDef *           channel0_port,
    uint16_t                 channel0_pin,
    GPIO_TypeDef *           channel1_port,
    uint16_t                 channel1_pin);
```

Reply:
Try `#include <stm32f3xx_hal.h>` in the Io_Efuse.h instead. I'm assuming the problem is that `Io_Efuse.h` needs `main.h` access to `GPIO_TypeDef` and `SPI_HandleTypeDef`. The .h file shouldn't need access to any of the constants so its better to keep it in the .c file.

Reply:
Let's just include main rn, because it's more portable that way. Including #include <stm32f3xx_hal.h> would break once we change mcu's